### PR TITLE
Notify bosh.io with checksum of the stemcell

### PR DESCRIPTION
**Pull request by mattcui**
For addressing [#126072163](https://www.pivotaltracker.com/story/show/126072163)
Already added a new task (notify-checksum-boshio) it in the pipeline, it's green -> http://bosh-softlayer-cpi.ci.bluemix.net/pipelines/bosh-softlayer-stemcell-community/jobs/notify-checksum-boshio/builds/17

@maximilien The code is ready for review and merge.

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -129,8 +129,8 @@ resources:
 - name: bosh-src
   type: git
   source:
-    uri: https://github.com/jianqiu/bosh.git
-    branch: 256.x-softlayer
+    uri: https://github.com/zhanggbj/bosh.git
```
Comment (maximilien) [inline ci/pipeline.yml:133]: Why is this still pointing to a private user repo?

File: ci/pipeline.yml [hunk 2]
```diff
@@ -213,23 +221,23 @@ resources:
 - name: bats
   type: git
   source:
-    uri: git@github.com:jianqiu/bosh.git
-    branch: 256.x-softlayer
+    uri: git@github.com:jianqiu/bosh-acceptance-tests.git
```
Comment (maximilien) [inline ci/pipeline.yml]: Same comment as above: "Why is this still pointing to a private user repo?"

File: ci/tasks/notify-checksum-boshio.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```
Comment (maximilien) [inline ci/tasks/notify-checksum-boshio.sh:1]: Why was this not implemented as a commend in the `sl_stemcells` executable CLI?

**Comment by maximilien** [comment]
@mattcui made some comments. I would like we address them. We can discuss tomorrow. If you need this merged and you agree with comments then let me know and I'll merge and add stories for each item. Thx.

**Comment by maximilien** [comment]
@mattcui you have not addressed my comments. If you cannot address them then add chores to do so later and I can merge this to expedite your needs.

File: ci/pipeline.yml (hunk 1 above)
Comment (mattcui) [reply ci/pipeline.yml:133]: For stemcell, currently, we still have to fork due to the following reasons:
1. Need to enable root ssh remote login, both Bluemix and eCPI require it.
2. For Bluemix stemcell, we need to install openssl 1.0.2 and fips module.

File: ci/pipeline.yml (hunk 2 above)
Comment (mattcui) [reply ci/pipeline.yml]: For stemcell, currently, we still have to fork due to the following reasons:

Need to enable root ssh remote login, both Bluemix and eCPI require it.
For Bluemix stemcell, we need to install openssl 1.0.2 and fips module.

File: ci/tasks/notify-checksum-boshio.sh (hunk 3 above)
Comment (mattcui) [reply ci/tasks/notify-checksum-boshio.sh:1]: We can if you think it's necessary, we could plan it in ecpi_v3. 
Opened a feature in Tracker -> https://www.pivotaltracker.com/story/show/130343579
Thanks.

**Comment by mattcui** [comment]
@maximilien please see my responses. Thanks.

File: ci/tasks/notify-checksum-boshio.sh (hunk 3 above)
Comment (maximilien) [reply ci/tasks/notify-checksum-boshio.sh:1]: Ok, thx

File: ci/pipeline.yml (hunk 1 above)
Comment (maximilien) [reply ci/pipeline.yml:133]: Ok

**Comment by maximilien** [comment]
Thx for adding [chore](https://www.pivotaltracker.com/story/show/130343579) for enhancement.

Best,

Max

**Pull request merged**